articles/adding-dark-theme-to-your-site/index.md

```diff
@@ -0,0 +1,178 @@
+The CSS variables for the default light theme are as shown below. You can change the color code to your favorite color and look attractive in both dark and light themes.
+
+The CSS styling for the default white theme is shown below.
+```CSS
+html{
+    --bg: #fff;
+    --bg-panel: #ebebeb;
+    --color-heading: rgb(27, 168, 14); 
+    --color-text: #333333;
+}
+```
+
+CSS styling for dark theme.
+```CSS
+html[data-theme='dark'] {
+    --bg: #333333;
+    --bg-panel: #434343;
+    --color-heading: #0077ff; 
+    --color-text: #B5B5B5;
+}
+```
+
+Next, we need to specify the CSS variables' use through the CSS file so that all the website elements will change when we click the toggle theme button.
+
+```css
+body { 
+    background-color: var(--bg);
+}
+
+.container {
+    background-color: var(--bg-panel);
+
+    h1{
+        margin: 0;
+        color: var(--color-heading)
+    }
+
+    p{
+        color: var(--color-text);
+       
+    }
+
+}
+```
+
+### Switch Theme Button
+In the next section, we will style our toggle switch which will be cheched to switch bewten the dark and ligh theme. The code for this toggle button cis as below. You can find the  code to this switch [here](https://codepen.io/mburnette/pen/LxNxNg)
+
+```css
+    input[type=checkbox] {
+        height: 0;
+        width: 0;
+        visibility: hidden;
+    }
+
+    label {
+        cursor: pointer;
+        text-indent: -9999px;
+        width: 52px;
+        height: 27px;
+        background: #1ba80e;
+        float: right;
+        border-radius: 100px;
+        position: relative;
+    }
+
+    label:after {
+        content: '';
+        position: absolute;
+        top: 3px;
+        left: 3px;
+        width: 20px;
+        height: 20px;
+        background: #fff;
+        border-radius: 90px;
+        -webkit-transition: 0.3s;
+        transition: 0.3s;
+    }
+
+    input:checked + label {
+        background: var(--color-heading);
+    }
+
+    input:checked + label:after {
+        left: calc(100% - 5px);
+        -webkit-transform: translateX(-100%);
+                transform: translateX(-100%);
+    }
+
+    label:active:after {
+        width: 45px;
+    }
+```
+
+### Adding the JavaScript
+We will handle the theme switching through JavaScript by changing the toggle switch's class name to either light or dark, as defined above. In doing so, we write two functions `changeThemeToDark()` and `chnageThemeToWhite()`.
+
+**Change the theme to dark.**
+```javascript
+    // Change theme to dark by adding the `dark` classname to html element.
+   const changeThemeToDark = () =>{
+    document.documentElement.setAttribute("data-theme", "dark")
+}
+
+```
+
+**Change the theme to Light.**
+```javascript
+    // Reset the html class to default
+   const changeThemeToDark = () =>{
+    document.documentElement.setAttribute("data-theme", "dark");
+}
+```
+
+### Using Local Storage to remember the theme.
```

Comment:
If possible, add 2 - 3 lines explaining what local storage is. (Technical aspects)